roxie/roxiemem/roxiemem.cpp

```diff
@@ -4763,7 +4763,7 @@ class CChunkingRowManager : public CRowManager
         }
         if (map)
             map->report(logctx, allocatorCache);
-        return peakPages;
+        return peakPages*HEAP_ALIGNMENT_SIZE;
```

Comment:
Will this be computed at 64-bits?

Reply:
no because HEAP_ALIGNMENT_SIZE is type memsize_t.

Reply:
I could add a cast to make it clearer.

Reply:
It's fine as is